### PR TITLE
DSND-2756: Rename statements to statement

### PR DESCRIPTION
**Pull request by kinpang-CH**
* Update private-api-sdk-java version
* Update pom.xml to dockerise Java 21 images

Resolves [DSND-2756](https://companieshouse.atlassian.net/browse/DSND-2756)

[DSND-2756]: https://companieshouse.atlassian.net/browse/DSND-2756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JAndrewCH — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml [hunk 1]
```diff
@@ -321,6 +320,21 @@
                 <artifactId>jib-maven-plugin</artifactId>
                 <version>${jib-maven-plugin.version}</version>
             </plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>${jib-maven-plugin.version}</version>
+                <configuration>
+                    <containerizingMode>packaged</containerizingMode>
```
Comment (cgregoryCH) [inline pom.xml:329]: Is a packaged mode necessary?

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml (hunk 1 above)
Comment (crowlandsCH) [reply pom.xml:329]: it's not as far as I can tell, have tested it locally without seems to work fine, is without packaged mode in my PR [here](https://github.com/companieshouse/psc-data-api/pull/120)

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml (hunk 1 above)
Comment (kinpang-CH) [reply pom.xml]: Changes merged from main has fixed this.

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by JAndrewCH — APPROVED** [review]

**Review by cgregoryCH — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**